[2025-07-15T04:40:27.801+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T04:40:27.815+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_first_run_and_custom_date.process_custom_date scheduled__2025-07-11T17:00:00+00:00 [queued]>
[2025-07-15T04:40:27.819+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_first_run_and_custom_date.process_custom_date scheduled__2025-07-11T17:00:00+00:00 [queued]>
[2025-07-15T04:40:27.819+0000] {taskinstance.py:2306} INFO - Starting attempt 6 of 6
[2025-07-15T04:40:27.825+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_custom_date> on 2025-07-11 17:00:00+00:00
[2025-07-15T04:40:27.828+0000] {standard_task_runner.py:63} INFO - Started process 3293 to run task
[2025-07-15T04:40:27.830+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_first_run_and_custom_date', 'process_custom_date', 'scheduled__2025-07-11T17:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpq2sldoqx']
[2025-07-15T04:40:27.832+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask process_custom_date
[2025-07-15T04:40:27.840+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-15T04:40:27.866+0000] {task_command.py:426} INFO - Running <TaskInstance: test_first_run_and_custom_date.process_custom_date scheduled__2025-07-11T17:00:00+00:00 [running]> on host bc01ce72dbd8
[2025-07-15T04:40:28.224+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_first_run_and_custom_date' AIRFLOW_CTX_TASK_ID='process_custom_date' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T17:00:00+00:00'
[2025-07-15T04:40:28.224+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T04:40:28.229+0000] {logging_mixin.py:188} INFO - --- Running python_callable: process_custom_date ---
[2025-07-15T04:40:28.229+0000] {logging_mixin.py:188} INFO - Successfully received custom date from params: {{ data_interval_end.subtract(months=1).end_of('month').to_date_string() }}
[2025-07-15T04:40:28.230+0000] {logging_mixin.py:188} INFO - For comparison, the DAG's logical date (ds) is: 2025-07-11
[2025-07-15T04:40:28.230+0000] {logging_mixin.py:188} INFO - --- End of process_custom_date ---
[2025-07-15T04:40:28.230+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-15T04:40:28.230+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T04:40:28.234+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_first_run_and_custom_date, task_id=process_custom_date, run_id=scheduled__2025-07-11T17:00:00+00:00, execution_date=20250711T170000, start_date=20250715T044027, end_date=20250715T044028
[2025-07-15T04:40:28.261+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-15T04:40:28.271+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T04:40:28.271+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
